webapp/layouts/default.vue

```diff
@@ -3,8 +3,8 @@
     <div class="main-navigation">
       <ds-container class="main-navigation-container" style="padding: 10px 10px;">
         <div>
-          <ds-flex class="main-navigation-flex" centered>
-            <ds-flex-item :width="{ lg: '3.5%' }" />
+          <ds-flex class="main-navigation-flex">
+            <ds-flex-item :width="{ lg: '30%' }" />
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-08-19T07:26:07Z" title="Monday, August 19th 2019, 9:26:07 am +02:00">Aug 19, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -4,7 +4,7 @@
       <ds-container class="main-navigation-container" style="padding: 10px 10px;">
         <div>
           <ds-flex class="main-navigation-flex">
-            <ds-flex-item :width="{ lg: '3.5%' }" />
+            <ds-flex-item :width="{ lg: '30%' }" />
```


This looks strange on my machine:
![Screenshot - 2019-08-19T092707 051](https://user-images.githubusercontent.com/2110676/63246610-5aa7ad80-c263-11e9-991b-fab63f56f5cc.png)

Reply:
> [<img alt="gha-gozu" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/gha-gozu) **Authored by [gha-gozu](https://github.com/gha-gozu)**
_<time datetime="2019-08-19T09:30:51Z" title="Monday, August 19th 2019, 11:30:51 am +02:00">Aug 19, 2019</time>_

---


I forgot to check the view on other devices. Will change it later today.